linkerd/app/Cargo.toml

```diff
@@ -31,7 +31,7 @@ http = "0.1"
 http-body = "0.1"
 hyper = "0.12"
 linkerd2-metrics = { path = "../metrics", features = ["test_util"] }
-linkerd2-proxy-api = { git = "https://github.com/linkerd/linkerd2-proxy-api", features = ["arbitrary"], tag = "v0.1.11" }
```

Comment:
Do we want to block this getting a linkerd2-proxy-api release out first?

Reply:
Yes we do

Reply:
@kleimkuhler We are using the released api now.